<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="UTF-8">

<title>MrBayes | index</title>
<link rel="stylesheet" href="css/style.css">
<link rel="stylesheet" href="css/handheld.css" media=
"only screen and (min-device-width:320px) and (max-device-width:568px)">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>

<body>
<header>
<h1>MrBayes: Bayesian Inference of Phylogeny</h1>


<nav>
<a href="./index.html" class="selected">Home</a> <a href=
"./download.html">Download</a> <a href="./manual.html">Manual</a>
<a href="./issues.html">Bug Report</a> <a href=
"./authors.html">Authors</a> <a href="./links.html">Links</a>
</nav>


<aside>
<a href="http://www.vr.se/"><img src="./img/vr.png" alt=
"Vetenskapsradet"></a> <a href="http://www.nrm.se/"><img src=
"./img/nrm.gif" alt="Naturhistoriska Riksmuseet"></a> <a href=
"http://nbis.se/"><img src="./img/nbis.svg" alt=
"National Bioinformatics Infrastructure Sweden (NBIS)"></a> <a href=
"http://www.nih.gov/"><img src="./img/nih.gif" alt=
"National Institutes of Health"></a> <a href=
"http://www.nsf.gov/"><img src="./img/nsf.gif" alt=
"National Science Foundation"></a>
</aside>
</header>


<main id="index">
<p>MrBayes is a program for Bayesian inference and model choice across
a wide range of phylogenetic and evolutionary models. MrBayes uses
Markov chain Monte Carlo (MCMC) methods to estimate the posterior
distribution of model parameters.</p>


<p>Program features include:</p>


<ul>
<li>A common command-line interface across Macintosh, Windows, and UNIX
operating systems;</li>


<li>Extensive help available from the command line;</li>


<li>Analysis of nucleotide, amino acid, restriction site, and
morphological data;</li>


<li>Mixing of data types, such as molecular and morphological
characters, in a single analysis;</li>


<li>Easy linking and unlinking of parameters across data
partitions;</li>


<li>An abundance of evolutionary models, including 4x4, doublet, and
codon models for nucleotide data and many of the standard rate matrices
for amino acid data;</li>


<li>Estimation of positively selected sites in a fully hierarchical
Bayesian framework;</li>


<li>Full integration of the BEST algorithms for the multi-species
coalescent.</li>


<li>Support for complex combinations of positive, negative, and
backbone constraints on topologies;</li>


<li>Model jumping across the GTR model space and across fixed rate
matrices for amino acid data;</li>


<li>Monitoring of convergence during the analysis, and access to a wide
range of convergence diagnostics tools after the analysis has
finished;</li>


<li>Rich summaries of posterior samples of branch and node parameters
printed to majority rule consensus trees in FigTree format;</li>


<li>Implementation of the stepping-stone method for accurate estimation
of model likelihoods for Bayesian model choice using Bayes
factors;</li>


<li>The ability to spread jobs over a cluster of computers using MPI
(for Macintosh (OS X) and UNIX environments only);</li>


<li>Support for the BEAGLE library, resulting in dramatic speedups for
codon and amino acid models on compatible hardware (NVIDIA graphics
cards);</li>


<li>Checkpointing across all models, allowing the user to seemlessly
extend a previous analysis or recover from a system crash;</li>
</ul>
</main>
</body>
</html>
